test/jdk/java/text/testlib/CollatorTestUtils.java

```diff
@@ -21,47 +21,42 @@
  * questions.
  */
 
-import java.lang.reflect.*;
-import java.util.Hashtable;
-import java.util.Enumeration;
-import java.util.Vector;
-import java.io.*;
-import java.text.*;
+import java.text.CollationElementIterator;
+import java.text.CollationKey;
+import java.text.Collator;
 
 /**
- * CollatorTest is a base class for tests that can be run conveniently from
- * the command line as well as under the Java test harness.
- * <p>
- * Sub-classes implement a set of methods named Test<something>. Each
- * of these methods performs some test. Test methods should indicate
- * errors by calling either err or errln.  This will increment the
- * errorCount field and may optionally print a message to the log.
- * Debugging information may also be added to the log via the log
- * and logln methods.  These methods will add their arguments to the
- * log only if the test is being run in verbose mode.
+ * CollatorTestUtils is a utility class that provides methods that
+ * Collator related tests can use.
  */
-public abstract class CollatorTest extends IntlTest {
+public final class CollatorTestUtils {
```

Comment:
Can the utilities in this class be merged into `TestUtils`?

Reply:
Merged CollatorTestUtils into TestUtils. I adjusted some method names which were previously in CollatorTestUtils to make more sense, now that they are in TestUtils.